doc/user/content/guides/cdc-mysql.md

```diff
@@ -0,0 +1,98 @@
+    ```
+
+### Deploy Debezium
+
+Debezium is deployed as a set of Kafka Connect-compatible connectors, so you first need to define a MySQL connector configuration and then start the connector by adding it to Kafka Connect.
+
+{{< note >}}
+Currently, Materialize only supports Avro-encoded Debezium records. If you’re interested in JSON support, please reach out in the community Slack or leave a comment on [this GitHub issue](https://github.com/MaterializeInc/materialize/issues/5231).
+{{</ note >}}
+
+1. Create a connector configuration file and save it as `register-mysql.json`:
+
+    ```json
+    {
+      "name": "your-connector",
+      "config": {
+          "connector.class": "io.debezium.connector.mysql.MySqlConnector",
+          "tasks.max": "1",
+          "database.hostname": "mysql",
+          "database.port": "3306",
+          "database.user": "user",
+          "database.password": "mysqlpwd",
+          "database.server.id":"223344",
+          "database.server.name": "dbserver1",
+          "database.history.kafka.bootstrap.servers":"kafka:9092",
+          "database.history.kafka.topic":"dbserver1.history",
+          "database.include.list": "db1",
+          "table.include.list": "table1",
+          "include.schema.changes": false
+        }
+    }
+    ```
+
+    You can read more about each configuration property in the [Debezium documentation](https://debezium.io/documentation/reference/connectors/mysql.html#mysql-connector-properties). By default, the connector writes events for each table to a Kafka topic named `serverName.databaseName.tableName`.
+
+### Create a source
+
+Debezium emits change events using an envelope that contains detailed information about upstream database operations, like the `before` and `after` values for each record. To create a source that interprets the [Debezium envelope](/sql/create-source/avro-kafka/#debezium-envelope-details) in Materialize:
+
+```sql
+CREATE SOURCE kafka_repl
+FROM KAFKA BROKER 'kafka:9092' TOPIC 'dbserver1.db1.table1'
+KEY FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY 'http://schema-registry:8081'
+VALUE FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY 'http://schema-registry:8081'
+ENVELOPE DEBEZIUM;
+```
+
+#### Transaction support
+
+Debezium provides [transaction metadata](https://debezium.io/documentation/reference/connectors/mysql.html#mysql-transaction-metadata) that can be used to preserve transactional boundaries downstream. We are working on using this topic to support transaction-aware processing in Materialize ([#7537](https://github.com/MaterializeInc/materialize/issues/7537))!
+
+### Create a materialized view
+
+Any materialized view defined on top of this source will be incrementally updated as new change events stream in through Kafka, as a result of `INSERT`, `UPDATE` and `DELETE` operations in the original MySQL database.
+
+```sql
+CREATE MATERIALIZED VIEW cnt_table1 AS
+SELECT field1,
+       COUNT(*) AS cnt
+FROM kafka_repl
+GROUP BY field1;
+```
```

Comment:
Maybe a Related pages section with a link to our doc on materialized views? Your call.

Reply:
I'll make a separate PR for improvements that affect both guides in terms of structure. I might bring the "Third Party" Debezium page back on that one, too. IMO, it helps raise visibility of Debezium within the docs and serves as a centralized page for everything CDC.